packages/dmn-js-drd/src/features/modeling/DrdUpdater.js

```diff
@@ -402,7 +403,7 @@ DrdUpdater.prototype.updateConnectionWaypoints = function(context) {
 
   if (edge) {
     edge.waypoints = map(connection.waypoints, function(waypoint) {
-      return drdFactory.createDiWaypoint(waypoint);
+      return drdFactory.createDiWaypoint(omit(waypoint, [ 'original' ]));
```

Comment:
I'd rather extract `x` and `y` coordinates here, only. Or is there any other waypoint specific information that we should take over?

Reply:
Cf. https://github.com/bpmn-io/bpmn-js/blob/develop/lib/features/modeling/BpmnFactory.js#L91